tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -50,6 +50,7 @@ const (
 	LB_NLB_TYPE                string = "net"
 	LB_ALB_TYPE                string = "app"
 	WAF_LOG_TYPE               string = "WAFLogs"
+	GUARDDUTY_LOG_TYPE         string = "GuardDutyLogs"
```

Comment:
I think this should be just `GuardDuty` to match the bucket key

Reply:
I'll defer to someone who uses guard duty who knows better here.